### DISCUSSION TITLE
Feature Request: use inlay hints to show parameters name of callable objects

### DISCUSSION THREAD

**Discussion by A-kirami** [microsoft/pylance-release#3127] [Enhancement]
Like `rust-analyzer`, this will be beneficial.

if support double-click inlay hints for parameters name, can be changed into keyword argument, that's even better.

![image](https://user-images.githubusercontent.com/66513481/182276421-91f233d1-2608-4ce8-ace3-f61626af553a.png)
![image](https://user-images.githubusercontent.com/66513481/182276453-717cc848-27f4-4aa8-aa39-dc34557947db.png)

**Comment by judej** [comment]
This has been implemented. locking thread. https://devblogs.microsoft.com/python/python-in-visual-studio-code-july-2022-release/#inlay-type-hints
